x-pack/plugin/core/src/main/resources/org/elasticsearch/xpack/profiler/index-template/profiling-symbols-private.json

```diff
@@ -0,0 +1,20 @@
+{
+  "index_patterns": [
+    ".profiling-symbols-private*"
+  ],
+  "composed_of": [
+    "profiling-symbols"
+  ],
+  "template": {
+    "settings": {
+      "index": {
+        "lifecycle.rollover_alias": "profiling-symbols-private"
```

Comment:
If I understand correctly, this index is not managed by ILM, if this is the case, why does it have a rollover alias?

Reply:
Good catch. That's a leftover. I'll fix this.